R/get_accessibility_2000.R

```diff
@@ -0,0 +1,45 @@
+#' Accessibility to Cities in 2000
+#'
+#' This resource provides global maps of travel time to cities in the year 2000.
```

Comment:
Could this mention that the reference is traveltime to cities >= 50,000 inhabitants? That's at least what I understand from skimming [JRC's docs](https://forobs.jrc.ec.europa.eu/gam).

Reply:
Yes it is travel time to cities of 50,000 or more people in year 2000. Can you clarify (I'm not very experienced with PRs) : you do it or I do it?

Reply:
You could add new commits to your branch that address these discussions here. This PR will be automatically updated.